tests/playTests.sh

```diff
@@ -499,6 +499,9 @@ $ZSTD --train-fastcover=k=56,d=8 && die "Create dictionary without input file"
 println "- Create dictionary with short dictID"
 $ZSTD --train-fastcover=k=46,d=8,f=15,split=80 "$TESTDIR"/*.c "$PRGDIR"/*.c --dictID=1 -o tmpDict1
 cmp tmpDict tmpDict1 && die "dictionaries should have different ID !"
+println "- Create dictionaries with shrink-dict flag enabled"
+$ZSTD --train-fastcover=steps=256 --shrink-dict=1 "$TESTDIR"/*.c "$PRGDIR"/*.c --dictID=1 -o shrinkDict
```

Comment:
e.g. will be `$ZSTD --train-fastcover=steps=256,shrink=1 <files> --dictID=1 -o shrinkDict` with the parsing parameters.

Also, don't need `--dictID=1`, thats an unnecessary parameter.